docs/specification/1.0.md

```diff
@@ -523,6 +524,36 @@ The following example illustrates a delete action:
 }
 ```
 
+
+
+#### overrideReason
+
+An **overrideReason** is described by the following attributes. No standard set of **overridereasons** exists today; implementers are encouraged to submit suggestions for standardization. To enable analysis of a card's usage, the CDS service SHALL send the same `code`, `codesystem`, and `label` across CDS Hooks requests for the same CDS client.
```

Comment:
Capitalize the first reference, but remove the formatting from the second:
An **OverrideReason** is described by the following attributes. No standard set of override reasons exists today;

Reply:
Instead of "No standard set of override reasons exists today", maybe "This specification does not prescribe a standard set of override reasons;